<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-D1GOMAD" failures="1" tests="1" name="test2" time="18.074" errors="0" timestamp="2021-11-09T10:52:06 IST">
  <testcase classname="com.qa.TC002_RecruitmentSearch_Duplicate" name="loginValidation" time="5.051">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.qa.TC002_RecruitmentSearch_Duplicate.loginValidation(TC002_RecruitmentSearch_Duplicate.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginValidation -->
</testsuite> <!-- test2 -->
